eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutablePrimitiveObjectHashMap.stg

```diff
@@ -744,13 +744,6 @@ final class Immutable<name>ObjectHashMap\<V> extends AbstractImmutable<name>Obje
         return this.delegate.chunk(size);
     }
 
```

Comment:
Hi @motlin, this should be in the other PR for `aggregateInPlaceBy`.

Reply:
I plan on landing the other one first. Afterwards, this will be a little easier to rebase.

Reply:
@motlin I merged the first PR if you want to rebase.